fla/ops/common/fused_chunk.py

```
@@ -0,0 +1,656 @@
+            b_gk = exp(b_g_last - b_g)
+            b_gn = exp(b_g_last)
+        if USE_G_GAMMA:
+            b_g_last = b_gamma * min(BT, T - i_t * BT)
+            b_gk = exp(b_g_last - b_g)
+            b_gn = exp(b_g_last)
+        if USE_G or USE_G_GAMMA:
+            b_gs = tl.where(m_s & m_t, exp(b_g[:, None] - b_g[None, :]), 0)
+            # [BT, BT]
+            b_s *= b_gs
+            # [BT, BV]
+            b_o = tl.dot(b_s.to(b_q.dtype), b_v) + tl.dot(b_q, b_h.to(b_q.dtype)) * b_gq[:, None]
```

Pylint (3.3.7)

[error] Possibly using variable 'b_gq' before assignment

(E0606)